src/model/guild/mod.rs

```diff
@@ -22,6 +22,8 @@ use tracing::error;
 #[cfg(all(feature = "model", feature = "cache"))]
 use tracing::warn;
 
+#[doc(hidden)]
+#[deprecated(note = "import the types from the `audit_log` module")]
```

Comment:
This doesn't trigger deprecation warnings because rustc currently ignores deprecated re-exports. It's more like comment here :).